src/util/message.c

```diff
@@ -110,13 +123,29 @@ void _singularity_message(int level, const char *function, const char *file_in,
     if ( level <= LOG ) {
         // Note __progname comes from the linker; the UID can be 5 characters and PID can be
         // 10-or-so characters.
-        char syslog_string[560]; // Flawfinder: ignore (512 max message length + 48 for header).
-        if (snprintf(syslog_string, 540, "%s (U=%d,P=%d)> %s", __progname, geteuid(), getpid(), message) >= 540) {
+        char log_message[560]; // Flawfinder: ignore (512 max message length + 48 for header).
+        if (snprintf(log_message, 540, "%s", message) >= 540) {
             // This case shouldn't happen unless we have a very strange __progname; nul-terminating to be sure.
-            syslog_string[559] = '\0';
+            log_message[559] = '\0';
+        }
+
+        // get current timestamp for file and stderr logging
+        time_t ltime;
+        ltime = time(NULL);
+        char *timestamp = asctime(localtime(&ltime));
+        // remove trailing newline
+        strtok(timestamp, "\n");
+
+        if (strcmp(singularity_config_get_value(LOG_SYSTEM), "syslog") == 0) {
+            syslog(syslog_level, "%s", log_message);
+        } else if (strcmp(singularity_config_get_value(LOG_SYSTEM), "std") == 0) {
+            fprintf(stderr, "%s: %s \n", timestamp, log_message);
```

Comment:
This will end up with some redundancy. While it does print LOG messages, it will also print all WARNING and ERROR messages as well (which also happens below). I would suggest (as I eluded to above) to make `std` actually `none`.